src/redux/books/books.js

```diff
@@ -1,52 +1,73 @@
+import axios from 'axios';
+
 const ADD_BOOK = 'ADD_BOOK';
 const REMOVE_BOOK = 'REMOVE_BOOK';
```

Comment:
- [ ] [OPTIONAL] You can use the `ducks pattern` as a value for the actions, this is a best practice and a requirement in the past milestone 
e.g.
              ```
              const ADD_BOOK = bookstore/books/ADD_BOOK
              const REMOVE_BOOK = bookstore/books/REMOVE_BOOK
              ```

Reply:
:rocket:  Alright
@Ismailco